ubccr/xdmod
# rawurlencode person name to avoid breaking XDMoD

## Description
Mostly just re-formatting to fit the current style guide. There was one line added to ensure that the `CCR.xdmod.ui.mappedPName` property was properly encoded before outputting it to javascript. 

## Motivation and Context
Encountered a Person name that included a `'` character. This broke the population of javascript properties in `index.php`. To resolve this problem the `rawurlencode` function was added to the affected data ( `rawurlencode` was used so that spaces were encoded to spaces as opposed to `+` which is how `urlencode` functions. ). 

## Tests performed
Manual Tests were performed 
- Associate a user with a person who has a `'` in their name.  
- Log in as that user
- Notice that the page does not load successfully
- clear the browser cookies
- Make the functional changes from this PR
- Log in as the user
- Notice that the page loads successfully and the name is displayed correctly.

## Types of changes


## Checklist:
